heat/core/types.py

```diff
@@ -347,55 +348,57 @@ def heat_type_of(obj):
     # last resort, type of the object at first position
     try:
         return canonical_heat_type(type(obj[0]))
-    except (KeyError, IndexError, TypeError,):
-        raise TypeError('data type of {} is not understood'.format(obj))
+    except (KeyError, IndexError, TypeError):
+        raise TypeError("data type of {} is not understood".format(obj))
 
 
 # type code assignment
-__type_codes = collections.OrderedDict([
-    (bool,    0),
-    (uint8,   1),
-    (int8,    2),
-    (int16,   3),
-    (int32,   4),
-    (int64,   5),
-    (float32, 6),
-    (float64, 7),
-])
+__type_codes = collections.OrderedDict(
+    [
+        (bool, 0),
+        (uint8, 1),
+        (int8, 2),
+        (int16, 3),
+        (int32, 4),
+        (int64, 5),
+        (float32, 6),
+        (float64, 7),
+    ]
+)
 
 # safe cast table
 __safe_cast = [
     # bool  uint8  int8   int16  int32  int64  float32 float64
-    [True,  True,  True,  True,  True,  True,  True,   True],  # bool
-    [False, True,  False, True,  True,  True,  True,   True],  # uint8
-    [False, False, True,  True,  True,  True,  True,   True],  # int8
-    [False, False, False, True,  True,  True,  True,   True],  # int16
-    [False, False, False, False, True,  True,  False,  True],  # int32
-    [False, False, False, False, False, True,  False,  True],  # int64
-    [False, False, False, False, False, False, True,   True],  # float32
-    [False, False, False, False, False, False, False,  True]   # float64
+    [True, True, True, True, True, True, True, True],  # bool
+    [False, True, False, True, True, True, True, True],  # uint8
+    [False, False, True, True, True, True, True, True],  # int8
+    [False, False, False, True, True, True, True, True],  # int16
+    [False, False, False, False, True, True, False, True],  # int32
+    [False, False, False, False, False, True, False, True],  # int64
+    [False, False, False, False, False, False, True, True],  # float32
+    [False, False, False, False, False, False, False, True],  # float64
 ]
 
 
 # same kind table
 __same_kind = [
     # bool  uint8  int8   int16  int32  int64  float32 float64
-    [True,  False, False, False, False, False, False,  False],  # bool
-    [False, True,  True,  True,  True,  True,  False,  False],  # uint8
-    [False, True,  True,  True,  True,  True,  False,  False],  # int8
-    [False, True,  True,  True,  True,  True,  False,  False],  # int16
-    [False, True,  True,  True,  True,  True,  False,  False],  # int32
-    [False, True,  True,  True,  True,  True,  False,  False],  # int64
-    [False, False, False, False, False, False, True,   True],   # float32
-    [False, False, False, False, False, False, True,   True]    # float64
+    [True, False, False, False, False, False, False, False],  # bool
```

Comment:
For more readability I would prefer to not change this formatting

Reply:
I do agree, but Black does not give us a chance to do this.